go.mod

```diff
@@ -1,21 +1,17 @@
-module github.com/mattermost/mattermost-plugin-starter-template
+module github.com/jwilander/mattermost-plugin-docup
 
-go 1.12
+go 1.14
 
 require (
 	github.com/blang/semver v3.6.1+incompatible // indirect
 	github.com/go-ldap/ldap v3.0.3+incompatible // indirect
+	github.com/go-redis/redis v6.15.7+incompatible // indirect
 	github.com/google/go-github v17.0.0+incompatible
-	github.com/hashicorp/go-hclog v0.9.2 // indirect
-	github.com/hashicorp/go-plugin v1.0.1 // indirect
-	github.com/lib/pq v1.1.1 // indirect
-	github.com/mattermost/go-i18n v1.11.0 // indirect
-	github.com/mattermost/mattermost-server v5.12.0+incompatible
-	github.com/pelletier/go-toml v1.4.0 // indirect
-	github.com/pkg/errors v0.8.1
-	github.com/stretchr/testify v1.3.0
+	github.com/mattermost/mattermost-server v5.11.1+incompatible
```

Comment:
Is this import sill needed?

Reply:
Thanks, it sneaked in somehow. It's not needed.